client/src/Components/JournalEntry/JournalEntry.js

```diff
@@ -3,9 +3,9 @@ import { Link } from "react-router-dom";
 import { DateTime } from 'luxon';
 import PropTypes from 'prop-types';
 import './JournalEntry.css';
-import { updateEntry } from "../../utilities/api-calls";
+import { fetchCall } from "../../utilities/api-calls";
 
-const JournalEntry = ({ journal, entryID }) => {
+const JournalEntry = ({ journal, entryID, updateLogs }) => {
```

Comment:
hey Joe! Would you mind adding `updateLogs` to  the propTypes? 🫶🏼 It looks like `entryId` could be added as well, (Maybe i missed this one?)

Reply:
@espressoGoddess For sure!! I'm not as familiar with PropTypes so would love to chat on this quickly before I push up that change